### PR TITLE
Fix problem with commitlint workflow file #79 #86

### LINKED ISSUE #79: Fix problem with commitlint workflow file
**Description**
[Commit lint workflow file](https://github.com/sivicstudio/starkludo/blob/dev/.github/workflows/commitlint.yml) currently has some problems preventing it from running successfully

**Action Items**
- Debug the commit lint workflow file
- Fix problem preventing it from running successfully
- _Check out the [commitlint docs](url) for help_

### PR DESCRIPTION
**Pull request by GoSTEAN**
## Description

Closes #79
- It fixes the commitlint  workflow now the file [commit lint workflow file](https://github.com/GoSTEAN/starkludo/blob/help/.github/workflows/commitlint.yml) doesn't have have any problem and it is now running successfully.

### CODE REVIEW

**Comment by ibsule** [comment]
Please update PR description according to the[ pull request template ](https://github.com/sivicstudio/starkludo/blob/dev/.github/pull_request_template.md)

@GoSTEAN

**Comment by GoSTEAN** [comment]
@princeibs I have updated the PR

File: client/.eslintrc.cjs
Comment (ibsule) [inline client/.eslintrc.cjs:1]: Please revert changes made to this file. The update you made is skipping all errors/warnings thereby making CI pass falsely.

File: client/package.json
Comment (ibsule) [inline client/package.json:1]: I noticed you removed the flag: `--max-warnings 0`. Can you add it back?

**Review by ibsule — CHANGES_REQUESTED** [review]

File: client/package.json
Comment (GoSTEAN) [reply client/package.json:1]: If I add it back it will make the workflow fail.

**Pull request merged**